modules/broadcast/src/backend/db.ts

```diff
@@ -172,29 +161,27 @@ export default class BroadcastDb {
       .then()
   }
 
-  // TODO: check naming
-  getOutboxCount(botId, schedule) {
-    return this.knex('broadcast_outbox')
-      .where({ botId, scheduleId: schedule['id'] })
-      .select(this.knex.raw('count(*) as count'))
-      .then()
-      .get(0)
+  async getOutboxCount(botId: string, schedule: Schedule): Promise<number> {
+    const result = await this.knex('broadcast_outbox')
+      .where({ botId, scheduleId: schedule.id })
+      .count<Record<string, number>>('* as qty')
+      .first()
+      .then(result => result!.qty)
+
+    // @ts-ignore
+    return parseInt(result)
```

Comment:
you only need to pass the radix param to make ts stop crying =

Reply:
Radix will not fix the issue, because technically it is already a number. The issue is because Postgres returns a number, while sqlite return a string. Made a minor change to clarify